crates/cxx-qt-gen/src/generator/cpp/inherit.rs

```diff
@@ -23,6 +23,7 @@ pub fn generate(
 
     for method in inherited_methods {
         let return_type = syn_type_to_cpp_return_type(&method.method.sig.output, cxx_mappings)?;
+        // TODO: we need to error if there is no base class and no qobject macro ?
```

Comment:
if inherit is used with no base class and no qobject macro, do we need to error?